src/hooks/progress/use-tutorial-progress-mutations/index.ts

```diff
@@ -44,6 +44,9 @@ const useTutorialProgressMutations = (): UseTutorialProgressMutationsResult => {
 				[TUTORIAL_PROGRESS_QUERY_ID, tutorialId, collectionId],
 				data
 			)
+
+			// Invalidate related collection progress
+			queryClient.invalidateQueries(['collectionProgress', collectionId])
```

Comment:
We want to invalidate any `use-collection-progress` queries for a tutorial progress update that involves a specific collection ID, as this new update affects collection progress.